filter-ssl/src/main/java/org/apache/mina/filter/support/SSLHandler.java

```diff
@@ -373,6 +374,9 @@ public void encrypt(ByteBuffer src) throws SSLException {
                 if (result.getHandshakeStatus() == SSLEngineResult.HandshakeStatus.NEED_TASK) {
                     doTasks();
                 }
+            } else if (result.getStatus() == SSLEngineResult.Status.BUFFER_OVERFLOW && bufferExpansionCount < 3) {
+            	outNetBuffer = SSLByteBufferPool.expandBuffer(outNetBuffer, outNetBuffer.capacity() * 2);
```

Comment:
Shall we format the code here?

Reply:
Fixed with 9588c80d07d47bd252ca366f28ffdc15803f088f